modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path

```diff
@@ -415,6 +415,11 @@
 		<td>//div[contains(@class,'dnd-th') and contains(text(),'Type')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>FOLDER_LABEL</td>
+		<td>//span[@class='lfr__object-web-view-object-definitions-title mb-0' and contains(text(),'${folderLabel}')]</td>
+		<td></td>
+	</tr>
```

Comment:
Let's wait for https://github.com/brianchandotcom/liferay-portal/pull/142465 to get merged